src/test/java/uk/gov/digital/ho/pttg/application/namematching/NameMatchingCandidatesServiceIT.java

```diff
@@ -23,6 +23,7 @@
         SpecialCharacters.class,
         AliasCombinations.class,
         EntireNonAliasName.class,
+        EntireLastNameAndEachFirstName.class,
         NamesWithFullStopSpaceCombinations.class
```

Comment:
Do we need a test for the ordering of the name combination generators?  Or is this implicitly tested in the BDD?